website/src/_posts/2022-09-3.0.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Uppy 3.0: Future proofed, fresh, loyally familiar" 
+date: 2022-09-01
+author: 
+  - aduh95
+  - arturi
+  - mifi
+  - murderlon
+image: ""
+published: true
+---
+
+<!--more-->
+
+## ESM
+
+Following the footsteps of many packages, we now only ship Uppy core and its plugins as ECMAScript Modules (ESM). On Uppy 2.x, we were shipping CommonJS.
+
+If are already using ESM yourself, or are using the CDN builds, nothing changes for you! If you are using CommonJS, you might need to add some tooling for everything to work, or you might want to refactor your codebase to ESM – refer to the [Pure ESM package gist](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) for added information and help on how to do that.
+
+<img class="border" alt="Uppy Compressor plugin showing compressed images notification" src="/images/blog/2.4-2.7/compressor.jpg">
+
+## Remote Sources
+
+We’ve introduced a new Uppy plugin/preset that combines Instagram, Facebook, Google Drive, Box, Unsplash, Dropbox, One Drive, Zoom and any future remote cloud sources that Uppy will support. It only works with Dashboard and let’s you enable all the above sources in only line:
+
+```js
+uppy.use(RemoteSoruces, { companionUrl: 'https://companion.uppy.io' }) // Remote Soruces need Companion server instance to work, you can get one hosted from Transloadit.com or self-host it
+```
```

Comment:
I think it would be nice to show a before and after code snippet to further highlight the improvement